json/patch.json

```diff
@@ -243,5 +243,10 @@
   "name": "7.29",
   "date": "2021-04-10T00:00:00.000Z",
   "id": 48
+ },
+ {
+  "name": "7.30",
+  "date": "2021-08-21T09:44:04.095Z",
+  "id": 49
```

Comment:
could we actually backdate this to approximately when the patch landed, and then run npm run build again?

Sorry it's so complicated

Reply:
fair enough